scripts/createAndApproveZendeskTicket.ts

```diff
@@ -54,10 +53,10 @@ const createAndApproveTicket = async () => {
     'OVERRIDE_EVENT_IDS'
   ])
   const ticketData = generateTicketData()
-  logger.info(`creating ticket with data ${JSON.stringify(ticketData)}`)
+  console.log('creating ticket')
```

Comment:
As this is a script we can keep the JSON stringify bit, which is probably quite useful

Reply:
Done